.vsts-ci.yml

```diff
@@ -113,13 +113,13 @@ extends:
           - powershell: New-Item -ItemType Directory -Path $(Build.SourcesDirectory)/artifacts/bin -Force
             displayName: Create artifacts/bin directory
           ${{ if and(eq(parameters.runTestBuild, false), ne(variables['Build.Reason'], 'PullRequest')) }}:
-            timeoutInMinutes: 90
+            timeoutInMinutes: 180
```

Comment:
Do you have numbers on how much longer the build is now due to the addition of the workloads?

Reply:
building the workloads themselves takes ~20mins, but signing is taking quite a lot more time. I believe we can optimize this since I see it signing all of the emsdk files again even though they are already signed.

Reply:
Because it's unpacking the content from the MSIs? Is it actually signing the files or just unpacking and then checking to see they are signed and then skipping them?

Reply:
I see them in the SigningRound* logs so I assume they're getting sent to ESRP. I haven't investigated yet why

Reply:
And the source nupkgs we're using from runtime/EMSDK, those should already be signed as part of the runtime build, correct? Is it possible we're pulling down the unsigned packages?

Reply:
The emsdk ones are always signed as part of its official build yes. For runtime they are signed either in it's current official build (when targetting a release branch) or in the VMR as part of one of the vertical builds.

Reply:
Signtool is supposed to be quite good now at avoiding re-signing. But I would expect re-signing in a build off main, since the source emsdk bits are unsigned.

Reply:
> since the source emsdk bits are unsigned.

I changed that already since I expected this to be a problem but it didn't help. Need to dig deeper.